app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -330,8 +311,9 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
             }
         }
 
-        viewLifecycleOwner.lifecycleScope.allowUndo(
-            requireView(), message,
+        MainScope().allowUndo(
+            requireActivity().getRootView()!!,
```

Comment:
Same as https://github.com/mozilla-mobile/fenix/pull/24398/files#r852355860

Reply:
Just to confirm, here we are switching to make the parent of the snackback to be the root of the `HomeActivity` that is alive even when the `BookmarkFragment` is gone?